spaces/domains.json

```
@@ -209,6 +209,7 @@
   "voting.font.community": "fontcommunity.eth",
   "snapshot.vswap.fi": "vbswap.eth",
   "vote.munchtoken.com": "munchproject.eth",
+  "vote.munchproject.io": "munchproject.eth",
```

Please keep only 1 subdomain.
```suggestion
  "vote.munchproject.io": "munchproject.eth",
```